python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1439,13 +1438,6 @@ def test_dataframe_transpose_category(num_cols, num_rows):
     got_function = gdf.transpose()
     got_property = gdf.T
 
-    # materialize our categoricals because pandas
```

Comment:
We don't need to circumvent this anymore, from "whats new in 1.0:" 
`Dtypes are now preserved when transposing a DataFrame where each column is the same extension dtype`